src/en/news/blog/2025/user-accounts-dashboard/index.md

```diff
@@ -0,0 +1,155 @@
+
+#### Apply Managed Policies to account user
+
+If the account user is non-root, there will be another option that will be visible which is **Manage Policies**.
+These policies give broad permissions for users:
+ - Full access versions (e.g. AmazonS3FullAccess) grant complete access to the service.
+ - Read-only versions limit actions to safe, non‑modifying operations.
+
+We can select the option based on the permission/access to be given.
+
+![Step 4.2: Create User: Apply managed policy](images/attach-managed-policy.png)  
+*Figure: Step 4.2 - Apply managed policy to account user*  
+
+After filling other required details we can click on create/edit user based on scenarios described above. Once the request completes you will be navigated back to user list page.
+
+Here, we can check the details of the user linked with an account, in the table `Account name` shows the user is linked with an account, and also if user is a root account user then it is depicted with a icon with username as seen below.  
+
+![Step 4.3: User Details: Account root user](images/account-root-user.png)  
+*Figure: Step 4.3 - Account root user*  
+
+We can also see more details of that user by going to user details section by expanding the table row.
+
+![Step 4.4: User details](images/user-details.png)  
+*Figure: Step 4.4 - Creating bucket level sync policy group.*  
+
+
+### 3.3 Link bucket with an Account
+
+Once the account user is created, we can now also link buckets with account, we can do this in following ways.  
+
+ - If a user has existing buckets prior to being associated with an account, those buckets will be automatically reassigned to the account upon linkage.
+ - User can create a bucket from the buckets page and also link it with the account.
+
+ > **NOTE** Account membership is permanent. Once added, users cannot be removed from their account.  
+
+Lets create a bucket and link it with an account. For this we will navigate to the Buckets page and click on `Create` button.
+
+we will get a form in which we need to enter the bucket name and owner details. At this point, we can check `Select account user` to link the bucket with an account.
+Now we can select the account user from the list and fill any other required details. Please note that the selected account user should have enough permission to create the bucket through managed policy.   
+
+
+![Step 5.1: Link bucket with account](images/link-bucket-with-account.png)  
+*Figure: Step 5.1 - Link user bucket with account.*   
+
+On submitting the form, on successful request completion you will be redirected to the bucket list page.  
+
+![Step 5.2: Bucket list page](images/bucket-list.png)  
+*Figure: Step 5.2 - Creating bucket level sync policy group.* 
+
+Similarly a bucket created with normal rgw user can be linked with the account by selecting already created bucket and then clicking on `Edit` to change the ownership of the bucket, we can follow the same process as we did above to do so.
```

Comment:
Similarly, a bucket created with a normal RGW user can be linked to an account by selecting the existing bucket and clicking on Edit to change its ownership. You can follow the same process as described above to do this.